alerts/cluster-network-operator/SouthboundStaleAlert

```diff
@@ -0,0 +1,23 @@
+# Southbound Stale Alert 
+
+## Meaning 
+
+The alert `SouthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
+without succesfully synching any changes to the Southbound Database.
+
+## Impact 
+
+Networking control plane is degraded and Networking Configuration updates applied to the cluster will not be applied.
+Existing workloads should continue to have connectivity, but the OVN-Kubernetes control plane and/or the OVN northbound database
+may not be functional. 
+
+## Diagnosis
+
+Either ovn is overloaded, i.e. the load on the cluster is too large due to a large number of services, network policies, pods, or nodes, or the control plane
+is degraded.
+
+## Mitigation 
+
+In the case that the cluster is overloaded, if the cluster is not on version 4.8, upgrading to that version should help as there have been numerous performance 
```

Comment:
Again, mitigation will depend on what scenario above failed. Depending on which scenario tested positive will then lead the mitigation. For each scenario, suggest a mitigation. You can keep it brief. Restarting pod is final mitigation.